src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -190,28 +190,28 @@
     <value>Mark assemblies with assembly version</value>
   </data>
   <data name="MarkAssembliesWithComVisibleTitle" xml:space="preserve">
-    <value>Mark assemblies with ComVisible</value>
+    <value>Mark assemblies with 'ComVisible'</value>
```

Comment:
💭 We should probably be consistent in titles regarding the use of `Attribute` suffix.